openapi.yaml

```diff
@@ -3612,21 +3613,25 @@ components:
                     description: The ID of the remote.
                     example: google-group-group:037m2jsg218b2wb
                     type: string
+                    x-speakeasy-ignore: true # deprecated for the remote_info object
                 remote_name:
                     description: The name of the remote.
                     example: Finance team
                     type: string
+                    x-speakeasy-param-suppress-computed-diff: true # cannot set this it's derived on the backend
                 group_type:
                     $ref: "#/components/schemas/GroupTypeEnum"
                     x-speakeasy-param-force-new: true
                 max_duration:
                     description: The maximum duration for which the group can be requested (in minutes).
                     type: integer
                     example: 120
+                    x-speakeasy-ignore: true
                 recommended_duration:
                     description: The recommended duration for which the group should be requested (in minutes). -1 represents an indefinite duration.
                     type: integer
                     example: 120
+                    x-speakeasy-ignore: true
```

Comment:
nixed because `recommended_duration` should come from the request_configuration. top level field is deprecated